juntagrico/admins/job_type_admin.py

```diff
@@ -11,10 +12,18 @@
 
 
 class JobTypeAdmin(BaseAdmin):
-    list_display = ['__str__']
+    list_display = ['__str__', 'activityarea',
+                    'duration', 'location', 'visible', 'last_used']
+    list_filter = ('activityarea', 'visible')
     actions = ['transform_job_type']
     inlines = [JobExtraInline]
 
+    def last_used(self, instance):
```

Comment:
das braucht es nicht, pro tätigkeitsbereich gibt es ja nicht hunderte von typen und der tätigkeitsbereichsleiter weiss ja wann er was zum letzen mal in etwa benutz hat

Reply:
Das ist hilfreich, wenn man alte Typen hat, die ähnlich heissen wie neue (z.B. 2 Varianten eines Jobs mit unterschiedlicher Dauer o.ä.).
Ich denke es sollte auch das Dropdown bei der Joberstellung danach gefiltert werden.
Zudem kommt es fast gratis.

Reply:
naja fast gratis finde ich für einen scan über alle job entitäteten welche an dem type hängen ein bisschen übertrieben.
Das dropdown sollte ganz klar alphabetisch sein. Das ist die logiste art wie leute suchen.
Wie gesagt, ein bereichskoordinator weiss wenn er einen jobtyp ändert anpasst oder einen nuen macht und den alten nicht mehr braucht. neu kann er den alten ja sogar unsichtbar machen.
Auch sehen die meisten bereichskoordinatoren nur ihre typen was eine handvoll ist. nur koordinatoren die auch bg mitgleder oder super admins sind sehen alle.

Reply:
Wir haben aktuell keine Bereichskoordinatoren (die Einsätze Ausschreiben) die nicht auch in der BG sind und die BG findets unübersichtlich.
Das unsichtbar machen hilft sicher, drum wollt ich das auch habe.
Diese Spalte hilft dabei zu entscheiden, welche Typen unsichtbar gemacht werden sollten und zeigt eine Information an, die sonst nirgendwo einfach ersichtlich ist.

Reply:
zum entscheiden ob ein typ unsichtbar gemacht werden soll oder nicht reicht es doch, dass der bereichsleiter seine typen durchgeht und entscheidet brauch ich noch oder brauch ich nicht mehr. ich sehe nicht weshalb man da dieses feld braucht welches sehr db rechen intensiv ist. 
und für eine gross angelegte aufräum aktion kannst du dir die info ja einmalig aus der db ziehen.
Da wir eh eine "Böhnli" aufräum story brauchen würde das meiner meinung nach eher da reingehören und dann eher mit anzahl einsätzen und löschen

Reply:
ich hab das mal hier spezifiert #163